khal/icalendar.py

```diff
@@ -62,6 +63,12 @@ def split_ics(ics, random_uid=False, default_timezone=None):
             tzs[key] = item
 
         if item.name == 'VEVENT':
+            if 'UID' not in item:
+                logger.warning(
+                    f"Event with summary '{items['SUMMARY']' doesn't have a unique ID,"
```

Comment:
```suggestion
                    f"Event with summary '{item['SUMMARY']}' doesn't have a unique ID. "
```

Reply:
Indeed. Dunno how I missed that. In my hurry I must have only run the tests on an earlier version where the summary wasn't yet logged... Thanks.